### DISCUSSION TITLE
mpv_animated脚本无法生成动图

### DISCUSSION THREAD

**Discussion by wli189** [dyphire/mpv-config#87] [General]
ffmpeg以配置于环境，但仍然无法生成动图
![image](https://github.com/dyphire/mpv-config/assets/32810655/3e2978d9-6cf1-4392-8199-a08a38fd8e61)

**Comment by dyphire** [comment]
尝试使用最新的 mpv_animated 脚本，有修复 vf 滤镜导致创建失败的错误
另外 ffmpeg 的 webp 编码器存在问题，个别视频还是会出现此错误，所以最新版默认格式改为了 gif
